README.md

```diff
@@ -147,28 +163,31 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
     are seeing this, run `kubectl get service frontend-external -o=yaml | kubectl apply -f-`
     to trigger load balancer reconfiguration.
 
-<!-- 
-### Option 3: Using Pre-Built Container Images 
+### Option 3: Using Pre-Built Container Images
+
+> 💡 Recommended if you want to deploy the app faster in fewer steps to an
+> existing cluster.
 
-> 💡 Recommended for test-driving the application on an existing cluster. 
+Follow "Option 1" or "Option 2" until you reach the `skaffold run` step. Instead
```

Comment:
Because the "recommendation" for this is "existing cluster," asking users to refer to option 1 or 2 for cluster setup is confusing IMO.  

Shouldn't users w/ existing clusters be able to go right to step 1 in this option 3?

Reply:
Valid point.